src/Service/MemoryInfoHandler.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "MemoryInfoHandler.h"
+
+#include "OrbitBase/Logging.h"
+
+namespace orbit_service {
+
+constexpr uint64_t kMemoryInfoProducerId = 1;
```

Comment:
we probably want one file with all reserved ids moved to somewhere more visible. Maybe to GrpcProtos/include/...

also move the starting id in ProducerSideServiceImpl.h to kExtranalProducerStartingId and set it to 1024 (or some other number >> 1)

Reply:
Make sense. 
I moved all reserved producer IDs to GrpcProtos/constants.h, including:
* `kLinuxTracingProducerId` used in LinuxTracingHandler.cpp
* `kMemoryInfoProducerId` used in MemoryInfoHandler.cpp
* `kExternalProducerStartingId` used as the starting id in ProducerSideServiceImpl.h

Thanks for the suggestion.